From 50f2556e150ba9f15809d2620689d93a30f606d0 Mon Sep 17 00:00:00 2001
From: Chris Lalancette <clalancette@gmail.com>
Date: Thu, 6 Jun 2024 13:54:12 -0400
Subject: [PATCH] Don't build testing or examples on the buildfarm.

This avoids the buildfarm running out of memory when
compiling.

Signed-off-by: Chris Lalancette <clalancette@gmail.com>
---
 rpm/template.spec.em | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index b1217d052964..3984d348c9a9 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -51,6 +51,8 @@ mkdir -p .obj-%{_target_platform} && cd .obj-%{_target_platform}
 %if !0%{?with_tests}
     -DBUILD_TESTING=OFF \
 %endif
+    -DBUILD_SOPHUS_TESTS=OFF \
+    -DBUILD_SOPHUS_EXAMPLES=OFF \
     ..
 
 %make_build
-- 
2.49.0

